src/index.js

```diff
@@ -160,7 +163,7 @@ function sanitizeHtml(html, options, _recursing) {
       stack.push(frame);
 
       var skip = false;
-      var hasText = frame.text ? true : false;
+      var hasText = !!frame.text;
```

Comment:
Not simply white space. Auto fix.

Reply:
Aha! Told everyone this convention wasn't just in my head 😆